www/index.html

```diff
@@ -51,11 +51,11 @@
     <!--
       The nav bar that will be updated as we navigate between views.
     -->
-    <ion-nav-bar class="bar-stable">
+    <ion-nav-bar class="bar-stable" ng-controller="NavBarCtrl">
       <ion-nav-back-button class="button-clear">
       </ion-nav-back-button>
       <ion-nav-buttons side='secondary'>
-        <button class='button'>
+        <button class='button' ng-click='modal.show()'>
           <i class='icon ion-ios-plus-outline'></i>
         </button>
       </ion-nav-buttons>
```

Comment:
Github won't let me comment below this. Down on line 70 I noticed an inline script. I'd recommend moving that to a file instead, though it certainly doesn't have to be a part of this PR.